recipe/meta.yaml

```diff
@@ -1,65 +1,86 @@
 {% set version = "1.5.5" %}
-{% set sha256 = "1fdfd5019a0b423259d91d77c9c8969bc3c2be59a3a13731904331a3257ac05f" %}
+{% set sha256 = "20d9451f59a608e70c93a21c22bc3081f9a1982d90ab50c2effb17fc5cbde577" %}
 
 package:
   name: cyclus
   version: {{ version }}
 
 source:
-  fn: cyclus-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/cyclus/cyclus/archive/{{ version }}.tar.gz
+  fn: cyclus-{{ version }}-{{ sha256 }}.zip
+  url: https://github.com/bennibbelink/cyclus/archive/refs/heads/pyinstall-cmake.zip
   sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 0
   skip: True  # [win or osx or py<=27]
   detect_binary_files_with_prefix: true
   preserve_egg_dir: True
     
 requirements:
   build:
     - cmake
+    - make
     - pkg-config
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-  host:
-    - coincbc <2.9.10
-    - boost-cpp <1.69
-    - libxmlpp
+    - coincbc
+    - libboost-devel
+    - libxmlpp-4.0
     - libxml2
+    - liblapack
     - hdf5
-    - glibmm
-    - glib 2.56
+    - sqlite
+    - glib
     - python
+    - pandas
     - jinja2
+    - cython
+    - pip
+    - setuptools
+  host:
+    - coincbc
+    - libboost-devel
+    - libxmlpp-4.0
+    - libxml2
+    - liblapack
+    - blis
+    - hdf5
+    - sqlite
+    - glib
+    - python
     - pandas
-    - cython  # [ not (py36 and osx) ]
-    - cython <0.28  # [ py36 and osx ]
-  run:
     - python
-    - {{ pin_compatible("boost-cpp", max_pin="x.x.x") }}
-    - {{ pin_compatible("coincbc", max_pin="x.x.x") }}
-    - sqlite
-    - hdf5
-    - libxmlpp
+    - pprintpp
+    - cython
+  run:
+    - coincbc
+    - libboost-devel
+    - libxmlpp-4.0
     - libxml2
-    - glibmm
-    - glib 2.56
+    - liblapack
+    - blis
+    - hdf5
+    - sqlite
+    - glib
+    - python
     - pandas
-    - jinja2
-    - pytables
+    - python
+    - pprintpp
 
 test:
   requires:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - nose
+    - pytest
+    - pytables
   source_files:
     - tests
+    - share/dbtypes.json
+    - src/hdf5_back_gen.py
 
 about:
   home: http://fuelcycle.org/
-  license: BSD 3-clause
+  license: BSD-3-clause
   license_file: LICENSE.rst
   summary: "The agent-based fuel cycle simulator."
 
```

Comment:
We should probably update the list if maintainers while we're here.

Reply:
Will do. Who should be added/removed?

Reply:
I added myself to the list of maintainers, I can make other changes if necessary